homeassistant/components/vicare/binary_sensor.py

```diff
@@ -124,7 +125,7 @@ async def _entities_from_descriptions(
     hass: HomeAssistant,
     entities: list[ViCareBinarySensor],
     sensor_descriptions: tuple[ViCareBinarySensorEntityDescription, ...],
-    iterables,
+    iterables: list[PyViCareDevice],
```

Comment:
I think this should be [`HeatingDeviceWithComponent`](https://github.com/somm15/PyViCare/blob/8ba411483a865e074d1146fd1b8b7a8c4f4be122/PyViCare/PyViCareHeatingDevice.py#L370)

Reply:
I think it even might be DeviceWithComponent, since this function receives both circuits, burners and compressors

Reply:
https://github.com/search?q=repo%3Asomm15%2FPyViCare%20HeatingDeviceWithComponent&type=code

Reply:
@joostlek now with PyViCarev2.28.1 in place [Compressors](https://github.com/somm15/PyViCare/blob/3a02a6467487a6999c2efaf31c31410d5003a25c/PyViCare/PyViCareHeatPump.py#L104), [HeatingCircuits](https://github.com/somm15/PyViCare/blob/3a02a6467487a6999c2efaf31c31410d5003a25c/PyViCare/PyViCareHeatingDevice.py#L378) and [GazBurners](https://github.com/somm15/PyViCare/blob/3a02a6467487a6999c2efaf31c31410d5003a25c/PyViCare/PyViCareGazBoiler.py#L268) all inherit from `HeatingDeviceWithComponent`